manifest.toml

```
@@ -0,0 +1,103 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "zabbix"
+name = "Zabbix"
+description.en = "Monitoring tool for diverse IT components, including networks, servers, VMs and cloud services"
+description.fr = "Outil pour monitorer des réseaux, des serveurs, des VMs et autres services en ligne"
+
+version = "5.0.41~ynh1"
+
+maintainers = []
+
+[upstream]
+license = "GPL-2.0-or-later"
+website = "https://www.zabbix.com"
+admindoc = "https://www.zabbix.com/manuals"
+code = "https://github.com/zabbix/zabbix"
+cpe = "cpe:2.3:a:zabbix:zabbix"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = ["amd64"]
+multi_instance = false
+ldap = true
+sso = true
+disk = "500M"
+ram.build = "200M"
+ram.runtime = "200M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/zabbix"
+
+    [install.init_main_permission]
+    help.en = "A public app doesn't need SSO auth : the auth page is opened for everyone"
+    help.fr = "Une application publique ne nécessite pas une authentification SSO : sa page d'authentication est ouverte à tous"
+    type = "group"
+    default = false
```

```suggestion
    default = "visitors"
```
 `false` is not a valid value for Bookworm